test/unit/index.js

```diff
@@ -37,7 +36,6 @@ async function do_test(array) {
     }
 }
 
-
```

Comment:
Really, please, no whitespace changes on any lines you're not modifying. It breaks git-blame for no reason.